runtime/near-wallet-contract/src/lib.rs

```diff
@@ -0,0 +1,35 @@
+#![doc = include_str!("../README.md")]
+
+use near_vm_runner::ContractCode;
+use once_cell::sync::OnceCell;
+use std::path::Path;
+
+/// Temporary (placeholder) Wallet Contract. Read from file once, then cache in memory.
+pub fn wallet_contract() -> &'static ContractCode {
+    static CONTRACT: OnceCell<ContractCode> = OnceCell::new();
+    CONTRACT.get_or_init(|| read_contract("wallet_contract.wasm"))
+}
+
+/// Temporary (placeholder) Wallet Contract that has access to all host functions from
+/// the nightly protocol. Read from file once, then cache in memory.
+pub fn nightly_wallet_contract() -> &'static ContractCode {
```

Comment:
In the implementation of `get_contract_code` this nightly variant is never used. Maybe instead of having two version of the `wallet_contract` function, there should only be one definition and the implementation will depend on the `nightly` compile-time feature flag.